packages/perseus/.eslintrc.js

```diff
@@ -17,8 +17,9 @@ module.exports = {
         // so feel free to update if you know what you're doing.
         "@typescript-eslint/no-restricted-imports": [
             "error",
-            "@khanacademy/perseus",
-            "@khanacademy/perseus-editor",
+            {
```

Comment:
That looks to match what's documented here: https://eslint.org/docs/latest/rules/no-restricted-imports#paths